clippy_lints/src/mem_replace.rs

```diff
@@ -62,7 +62,7 @@ declare_clippy_lint! {
     /// }
     /// ```
     ///
-    /// The [take_mut](https://docs.rs/take_mut) crate offers a sound solution,
+    /// The [take_mut] crate offers a sound solution,
```

Comment:
```suggestion
    /// The [`take_mut`] crate offers a sound solution,
```

Reply:
Yup, `take_mut` isn't in scope, good catch.